crowbar_framework/app/models/api/backup.rb

```diff
@@ -174,6 +174,9 @@ def create_archive
     self.version = ENV["CROWBAR_VERSION"]
     self.size = path.size
     self.migration_level = ActiveRecord::Migrator.current_version
+  rescue Exception => e
```

Comment:
Yeah, I updated this but I'm not sure if this is still not too wide. @jsuchome, @MaximilianMeister any hints on this?